europe/finland.md

```diff
@@ -6,7 +6,6 @@ Yggdrasil configuration file to peer with these nodes.
 * Tuusula, Hetzner, operated by [cwinfo](https://cwinfo.net) and [cwchristerw](https://christerwaren.fi)
   * `tls://fi1.servers.devices.cwinfo.net:61995`
   * `tls://65.21.57.122:61995`
-  * `tls://[2a01:4f9:c010:664d::1]:61995`
```

Comment:
@cwchristerw

Reply:
@offblacc Was correct because new IPv6 is now 2a01:4f9:c012:a8df::1 because Hetzner wasn't supporting that old IPv6 subnet when transferring server to different account. I have decided to start soon use waren.dev domain for previous cwinfo.net servers at some point.

Reply:
It's possible that I'm moving away from Hyperboria and Yggdrasil, and choose to use more private networking solution. I have mainly used Yggdrasil as way to connect to host that don't have static IPv4 and/or IPv6.